tests/test_pytype.py

```diff
@@ -0,0 +1,15 @@
+import subprocess
+import sys
+import pytest
+
+
+@pytest.mark.skipif(
+    sys.version_info >= (3, 13), reason="Need python < 3.13 for pytype"
+)
+def test_pytype():
+    result = subprocess.run(
+        ["pytype", "vulture/core.py", "--disable=attribute-error"],
+        capture_output=True,
+        text=True,
+    )
+    assert result.stdout.strip().endswith("Success: no errors found")
```

Comment:
Let's check whether the exitcode is 0 to make this more robust.

Reply:
@jendrikseipp Oh the `check=True` does that. If the exitcode isn't 0, subprocess will raise a CalledProcessError.

Reply:
I think it's best to make that check explicit and to remove the string comparison (and to remove the then obsolete kwargs from the run call).